src/main/java/meditracker/MediTracker.java

```diff
@@ -56,20 +58,26 @@ public void run() throws MediTrackerException, ArgumentNotFoundException {
         while (!isExit) {
             String fullCommand = Ui.readCommand();
             Ui.showLine();
-            Command command = Parser.parse(fullCommand);
-            command.execute(medicationManager, dailyMedicationManager);
-            isExit = command.isExit();
+            try {
+                Command command = CommandParser.parse(fullCommand);
+                command.execute(medicationManager, dailyMedicationManager);
+                isExit = command.isExit();
```

Comment:
Might be better to scope this outside of the try catch block as it will not throw any of the exceptions below

Reply:
I think the exception can be thrown in the parse and execute method? I tried to scope them out but invalid input will lead to runtime error.

Reply:
I meant moving line 64 outside of the try catch block

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/75/commits/e7725fc104ce3e47f2b5cb1594b3a710c42b0518